eng/pipelines/common/templates/runtimes/build-test-job.yml

```diff
@@ -89,9 +89,13 @@ jobs:
       - ${{ if eq(parameters.runtimeFlavor, 'coreclr') }}:
         - name: liveRuntimeBuildParams
           value: ${{ format('clr.corelib+libs.ref+libs.native -rc {0} -c {1} -arch {2} -ci', coalesce(parameters.liveRuntimeBuildConfig, parameters.buildConfig), parameters.liveLibrariesBuildConfig, parameters.archType) }}
+        - name: liveRuntimeArtifactsPathArg
+          value: ${{ format('/p:CoreCLROverridePath={0}/artifacts/bin/coreclr/{1}{2}.{3}.{4}', $(Build.SourcesDirectory), parameters.osGroup, parameters.osSubgroup, parameters.archType, coalesce(parameters.liveRuntimeBuildConfig, parameters.buildConfig)) }}
       - ${{ if eq(parameters.runtimeFlavor, 'mono') }}:
         - name: liveRuntimeBuildParams
           value: ${{ format('mono.corelib+libs.ref+libs.native -rc {0} -c {1} -arch {2} -ci', coalesce(parameters.liveRuntimeBuildConfig, parameters.buildConfig), parameters.liveLibrariesBuildConfig, parameters.archType) }}
+        - name: liveRuntimeArtifactsPathArg
+          value: ${{ format('/p:MonoOverridePath={0}/artifacts/bin/mono/{1}{2}.{3}.{4}', $(Build.SourcesDirectory), parameters.osGroup, parameters.osSubgroup, parameters.archType, coalesce(parameters.liveRuntimeBuildConfig, parameters.buildConfig)) }}
```

Comment:
Curious and to understand, why is this needed now? 

We didn't use the runtime to build the tests?

Reply:
This is needed now because there's no alternative to specify a different path to the CoreCLR/Mono artifacts for System.Private.CoreLib without overriding the whole path (unlike Libraries, which has overrides for various different paths that we can do when we set TargetOS=AnyOS)

Reply:
With https://github.com/dotnet/runtime/pull/61754 or some equivalent PR, we can remove this since we won't be referencing System.Private.CoreLib directly.

Reply:
Ah, ok, makes sense.